client/blocks/image-editor/image-editor-canvas.jsx

```diff
@@ -100,9 +101,9 @@ export class ImageEditorCanvas extends Component {
 	}
 
 	fetchImageBlob( src ) {
-		const { siteSlug, isPrivateAtomic } = this.props;
+		const { siteSlug, isJetpack } = this.props;
 		const { filePath, query, isRelativeToSiteRoot } = mediaURLToProxyConfig( src, siteSlug );
-		const useProxy = isPrivateAtomic && filePath && isRelativeToSiteRoot;
+		const useProxy = ! isJetpack && filePath && isRelativeToSiteRoot;
```

Comment:
For consistency purposes, we should either cast this to boolean using `Boolean` or cast the `filePath` with `!!` like I'm suggesting [here](https://github.com/Automattic/wp-calypso/pull/90895/files#r1612421024), or the expression return type will be `bool | ".` Functionally it doesn't make a differently as "is truth, but it could cause confusion.